mailserver/mailserver.go

```diff
@@ -288,24 +290,56 @@ func (s *WMailServer) DeliverMail(peer *whisper.Peer, request *whisper.Envelope)
 	}
 
 	log.Debug("Processing request",
-		"lower", lower, "upper", upper,
+		"lower", lower,
+		"upper", upper,
 		"bloom", bloom,
 		"limit", limit,
 		"cursor", cursor,
-		"batch", batch)
+		"batch", batch,
+	)
 
 	if batch {
 		requestsBatchedCounter.Inc(1)
 	}
 
-	_, lastEnvelopeHash, nextPageCursor, err := s.processRequest(
-		peer,
-		lower, upper,
+	iter := s.createIterator(lower, upper, cursor)
+	defer iter.Release()
+
+	bundles := make(chan []*whisper.Envelope, 5)
```

Comment:
is it just to avoid blocking or this number means something else?

Reply:
Just avoid blocking. Pretty arbitrary.